FE/todo-app/src/types/Auth.ts

```diff
@@ -27,6 +27,12 @@ export type RegisterResponse = {
   };
 };
 
+export type MeResponse = {
+  id: number;
+  username: string;
+  email: string;
+};
```

Comment:
co si vybavuju tak response je mnohem větší jak 3 řádky. Dávej všude kompletní responsy, a ne jen to co se ti líbí.